File: ejercicios/python/ejercicio065.py
def potencia_recursiva(base, exponente):
   if (exponente == 0):
      return 1
   else:
      return (base * potencia_recursiva(base, exponente-1))

if __name__ == "__main__":
   base = int(input("ingrese la base : "))
   exponente = int(input("ingrese el exponente : "))
   if (exponente >= 0):
      print (str(base) + " elevado a " + str(exponente) + " es : " + str(potencia_recursiva(base, exponente)))
   else:
      print("Error en exponente " + str(exponente) + " debe ser positivo.")
      

File: ejercicios/c++/ejercicio065.cpp
#include <iostream>
using namespace std;

long potencia_recursiva(int base, int exponente){	if(exponente == 0)		return 1;	else		return base * potencia_recursiva(base, exponente-1);}

int main() {	int base, exponente;	cout << "ingrese la base : ";	cin >> base;	cout << "ingrese el exponente : ";	cin >> exponente;	if (exponente >= 0)		cout << base << " elevado a " << exponente << " es : " << potencia_recursiva(base, exponente) << endl;	else		cout << "Error en exponente " << exponente << " debe ser positivo." << endl;	return 0;}
